R/simulate_tools.R

```diff
@@ -290,6 +290,8 @@ spim_simulate_remove_dates_to <- function(obj, date) {
     return(obj)
   }
 
+  date <- date - 1
```

Comment:
Can we make an issue to document this and check that is definitely correct, seems like the sort of thing that will cause problems in future